### ISSUE TITLE
pydot would not work until I installed graphviz on macOS 

### ISSUE THREAD

**Issue by nathan-cummings-ukaea** [trungdong/prov#141]
### Steps

- I have a small prov script that I know works
- On a system that DOESN'T have prov installed, running the script resulted in:
`ModuleNoteFoundError: No module named 'prov'`
as expected.

- I ran:
`pip3 install "prov[dot]"`
(in quotes because of how zsh handles '[]')
- Running the script again, I get:
`FileNotFoundError: [Errno 2] "dot" not found in path`

After a bit of searching, I found that installing graphviz may be necessary, so I ran:
`brew install graphviz`
which came with a number of dependencies of its own.

The script was then able to run correctly and produce the graph.

Could the graphviz dependancy be addressed in the package?

Many thanks

**Comment by nathan-cummings-ukaea** [comment]
I recreated the steps on another Mac that had not had the prov package or graphviz installed before and verified that exact same issues occurred with the same steps taken to resolve.

**Comment by trungdong** [comment]
You are right, the `[dot]` option only install the required Python package to allow exporting the "dot" format. [Graphviz](https://graphviz.org) is required to convert the exported dot representation to a graphical format, like PNG or SVG.

Unfortunately, Graphviz cannot be installed via Python. As you found, you can install it with homebrew on Mac (which does not come installed as default, BTW). On Windows or Linux, installation instructions would be different.

In later releases, I will make it clear about the requirement of Graphviz in some use cases and point users to https://graphviz.org/download/ for installation instruction.
